src/WinRT.Runtime/Projections/IList.net5.cs

```diff
@@ -1250,7 +1250,7 @@ private static uint GetManyHelper(global::System.Collections.Generic.IList<T> so
         public static readonly IntPtr AbiToProjectionVftablePtr;
         static IList()
         {
-            if (IListMethods<T>.AbiToProjectionVftablePtr == default)
+            if (RuntimeFeature.IsDynamicCodeCompiled && IListMethods<T>.AbiToProjectionVftablePtr == default)
             {

```

Comment:
This and a couple other changes similar to this probably needs to be updated to reflect the findings that this conditional isn't compatible with the linker.  But it also looks like the other PR does fix this, so I am thinking of leaving this for now in this PR and having the other PR fix them the correct way when that conflict is resolved.  Thoughts?

Reply:
Yeah I think let's just leave this as is and then once this is merged and #1395 is merged, I can just rebase #1429 and do another pass on all runtime feature switches and fix any new occurrences from this PR as well. Will also make things easier to review since this PR wouldn't also be moving things around to change those switches 🙂